src/schema/v2/__tests__/deleteUserMutation.test.ts

```diff
@@ -56,7 +56,7 @@ describe("Delete a user", () => {
             new HTTPError(
               `https://stagingapi.artsy.net/api/v1/user/bats? - {"error":"User Not Found"}`,
               404,
-              `{"error":"User Not Found"}`
+              { error: "User Not Found" }
```

Comment:
I left all the specs in the repo alone (some passing `'Something Went Wrong'` strings, and some passing objects), except for specs that were clearly stringifying JSON from Gravity - testing that out in practice today, and that's no longer accurate - `body` is coming in as JSON and not stringified.

Reply:
Whoops, I called it too early. There's actually a bunch of specs that need updating to correctly stub an error.